### DISCUSSION TITLE
页面底部参考文献不能裂开，导致有留白

### DISCUSSION THREAD

**Discussion by Kelipapa** [obster-y/XJTU-thesis#79] [Q&A]
![image](https://user-images.githubusercontent.com/106446236/171148518-84737c26-9921-4020-87a8-a0fb573caac6.png)

答辩老师提的

word就没问题

![image](https://user-images.githubusercontent.com/106446236/171148881-3dbc7b81-ca48-4dde-a102-1a753c05e09b.png)

求问，急

**Comment by Kelipapa** [comment]
另外，还是行距的问题，Word模板里，全是中文文献的话，一页可以排42行
但如果全是英文文献，可以排47行
现在LaTeX里英文文献只能排42行（如果解决上面的问题）

![image](https://user-images.githubusercontent.com/106446236/171157138-d76a6336-852d-4f15-aca1-95fa5236055f.png)
![image](https://user-images.githubusercontent.com/106446236/171157410-033aa5d9-a997-470f-b70b-7ead94d0f866.png)

**Reply by obster-y** [reply]
第二个问题无法解决，因为 Word 的行高除了和字号有关，还和字体有关，如果答辩老师还有问题，只能说明他不懂还胡乱提要求；如果非要做到和 word 一模一样，那唯一的解决方法就是换 Word。
